packages/gatsby-theme-blog-core/src/components/post-hero.js

```diff
@@ -0,0 +1,15 @@
+import React from "react"
+import Image from "gatsby-image"
+
+const Hero = ({ post }) => (
+  <>
```

Comment:
NB: I don't think the fragment is necessary here, if the next line resolves to false, it shouldn't render anything

Reply:
React doesn't seem to link it without the fragment. I was going to dig into why, but seemed like it wasn't worth it for now.